cast/src/main.rs

```diff
@@ -56,7 +56,6 @@ async fn main() -> Result<()> {
 
     // todo: #2052 take network from scarb config if flag not provided
     let network_name = cli.network.unwrap_or_else(|| {
-        // todo: #2107
```

Comment:
How about we added `--machine-readable` flag as well?

Reply:
yes, this is one possibility and perhaps mvp scenario. we could take a look at foundry/scarb/come other tools to see how it is handled there.

can be done under separate issue in different PR if you wish

Reply:
https://github.com/software-mansion/protostar/issues/2146 I would take it just after this PR